### DISCUSSION TITLE
Can one set of styles apply to same files in multiple locations?

### DISCUSSION THREAD

**Discussion by arelkin** [openstyles/stylus#1350] [Q&A]
I have local versions of web pages and the same pages on a server

file:////Volumes/Resources/tools/verify_data.html
https://myurl.com/tools/verify_data.html

is there a way for one set of styles to apply to both of these?

**Comment by tophf** [comment]
Just add both URLs.
https://github.com/openstyles/stylus/wiki/Writing-styles#applying-styles-to-specific-sites

![](https://user-images.githubusercontent.com/1388389/43850913-8246c6f2-9b39-11e8-8d66-ce477e3c000d.png)

```css
@-moz-document domain("images.example.com"),
               domain('imagehost.com'),
               url-prefix(https://example.com/images),
               url(https://www.example.com/test.html) {
    /*
        The code in here only applies to:
            - Pages on the domains images.example.com and imagehost.com
            - Pages whose URLs start with https://example.com/images
            - The page https://www.example.com/test.html
    */
}
```
